homeassistant/components/roborock/switch.py

```diff
@@ -75,15 +77,26 @@ async def async_setup_entry(
     coordinators: dict[str, RoborockDataUpdateCoordinator] = hass.data[DOMAIN][
         config_entry.entry_id
     ]
+    possible_entities = [
```

Comment:
It would be nice if don't need to create the entity before calling the update function. One benefit would be that we could pass the result of the update to the entity when creating it so we don't need to update twice.